prow/config.yaml

```diff
@@ -78,6 +78,38 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  kubernetes/dns:
+  - name: integration-test
+    agent: kubernetes
+    context: integration-test
+    branches:
+    - master
+    rerun_command: "/test integration-test"
+    trigger: "(?m)^/test( all| pull-community-verify),?(\\s+|$)"
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/google_containers/busybox
```

Comment:
I'll assume you'll need at least run bootstrap.py with one of our base images (kubekins, or gcloud-in-go), and have `--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)` to checkout the PR of your repo